src/output.rs

```diff
@@ -28,23 +30,27 @@ impl OutputType {
             .or_else(|_| env::var("PAGER"))
             .unwrap_or(String::from("less"));
 
-        let less_path = PathBuf::from(&pager);
+        let pagerflags = shell_words::split(&pager).unwrap_or(vec![pager]);
+
+        let less_path = PathBuf::from(&pagerflags[0]);
```

Comment:
What happens if we pass `BAT_PAGER=""`? Does this panic?

Previously, setting `BAT_PAGER=""` would disable the pager. I'd like to keep this behavior.

Reply:
Fixed.